spec/beaker/hypervisor/vsphere_spec.rb

```diff
@@ -58,6 +58,20 @@ module Beaker
 
       end
 
+      it 'does not provisions hosts if provision is set to false' do
+        MockVsphereHelper.powerOff
+        hosts = make_hosts()
+        hosts[0]["provision"] = false
+        vsphere = Beaker::Vsphere.new( hosts, make_opts )
+
+        vsphere.provision
+
+        hosts.each do |host|
+          expect( MockVsphereHelper.find_vm( host.name ).powerState ) == "poweredOff"
```

Comment:
ahh, thanks. That does indeed what I would expect from my previous code.

Please note that existing tests are also wrong and should be fixed as well.